app/controllers/cards_controller.rb

```diff
@@ -39,7 +38,20 @@ def destroy
     redirect_to cards_path
   end
 
+  def check
+    @card = Card.find(params[:card][:id])
+    
+    if @card.check_word(params[:card][:translated_text],params[:answer])
+      flash[:success] = "Right"
+      redirect_to root_url
+    else
+      flash[:info] = "Wrong"
+      render 'home/index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.